src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -65,13 +84,53 @@ private static object HandleClass(Type memberType, int levelsLeft)
     private static Array HandleSingleDimensionalArray(Type arrayType) =>
         Array.CreateInstance(arrayType.GetElementType(), 0);
 
+    private static object HandleString(string memberName)
+    {
+        if (s_jsonData == null)
+        {
+            return $"{memberName}_val";
+        }
+
+        foreach (var pair in s_jsonData)
+        {
+            var key = pair.Key;
+            if (memberName.Contains(key, StringComparison.InvariantCultureIgnoreCase))
+            {
+                var data = ((JArray)pair.Value).ToObject<string[]>();
+                return data[memberName.Length % data.Length];
+            }
+        }
+
+        return $"{memberName}_val";
+    }
+
+    private static object HandleSystemType(Type systemType, string memberName) =>
+        systemType.FullName switch
+        {
+            "System.DateTime" => new DateTime(memberName.Length * 100 % int.MaxValue, memberName.Length % 12, memberName.Length % 30, memberName.Length % 24, memberName.Length % 60, memberName.Length % 60),
```

Comment:
Lets minimize the modulo operations. Maybe just set month, day, and hour.

Reply:
I set year, month, and day